get-compose-running.sh

```diff
@@ -21,6 +21,15 @@ containers_running() {
         healthy=$(docker ps -q --filter "id=${id}" --filter "health=healthy" | wc -l)
         is_ok=$((noHealth + healthy))
         num_ok=$((num_ok + is_ok))
+        is_dead=$((1 - noHealth + healthy))
+
+        if [ ${is_dead} -eq 1 ]; then
+            consumer_name=$(docker ps -a --filter="id=1992a8399583" --format="{{ .Names }}")
+            echo -e "\nERROR: Container '$consumer_name' is DEAD\n"
+            docker logs ${id}
```

Comment:
I think this could be improved limiting the output to a fixed number of lines, because some containers could flood the output.

Reply:
Usually the crash output is short, because it is a crash in the starting phase. Limiting the output can produce a lack of visibility.

Reply:
Then I suggest a wide limit, what about `--tail 100`?